src/views/Card.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { NavLink } from 'react-router-dom';
+import { useParams } from 'react-router-dom/cjs/react-router-dom.min';
+import { getCardbyId } from '../services/cards';
+
+export default function Card() {
```

Comment:
I would recommend adding some loading state, as an empty page will flash on the screen before the data is fetched